code/controlconfig/controlsconfig.cpp

```diff
@@ -2889,8 +2889,8 @@ int check_control_used(int id, int key)
 	// special case to allow actual mouse wheel to work with trigger controls --wookieejedi
 	if (item.type == CC_TYPE_TRIGGER) {
 
-		int first_btn = 1 << item.first.get_btn();
```

Comment:
int first_btn = (item.first.get_cid() == CID_MOUSE) ?  item.first.get_btn() : -1;
int second_btn = (item.second_btn.get_cid() == CID_MOUSE) ? item.second.get_btn() : -1;

Reply:
Good question, though this does not likely cover both cases of `Use_mouse_to_fly` that `mouse_get_btn_down` accounts for?